### ISSUE TITLE
[Feature Request]: individual tags for text strings

### ISSUE THREAD

**Issue by swift2512** [grokability/snipe-it#15707]
### Is your feature request related to a problem? Please describe.

Don't use tags made from other tags because what works for English, doesn't work for majority of other languages. I'm focused on import at the moment, so I'll use it as example. In screenshot bellow, not only capitalization is wrong, but also forms doesn't match. It looks terrible and very amateurish.
![image](https://github.com/user-attachments/assets/32181dab-9565-4173-bb34-b593fa3bf9a2)

### Describe the solution you'd like

If product is multi-liungual, text strings should be language neutral and constructed in the way where it's easy to get proper sounding expression. If you can't rearrange order of words, you have a problem as translator.

Basically, it's 'view blade' titles for lists and individual items, plus strings in import forms. I would say, it's less than 50 extra strings and it wouldn't have impact on site load times and performance. Also, some strings need to have text line when there's only 1 item. For example, notification about items bellow threshold - even in English it doesn't sound or look OK.
![image](https://github.com/user-attachments/assets/48cd1b92-5e39-442f-89f0-f3505c637645)

**Comment by snipe** [comment]
> but also forms doesn't match. It looks terrible and very amateurish.

You know you can open issues without being rude, right?

We re-use those language strings to prevent translators from having to re-translate what looks like the same thing over and over. As I'm sure you've noticed, we have been transitioning to using pluralized strings more often, but it's a process to update everything.
